packages/core/src/config.js

```diff
@@ -52,44 +53,22 @@ const getAppId = () => {
 };
 
 const getSocketURL = () => {
-    let server_url = window.localStorage.getItem('config.server_url');
-    if (!server_url) {
-        // const to_green_percent = { real: 100, virtual: 0, logged_out: 0 }; // default percentage
-        // const category_map     = ['real', 'virtual', 'logged_out'];
-        // const percent_values   = Cookies.get('connection_setup'); // set by GTM
-        //
-        // // override defaults by cookie values
-        // if (percent_values && percent_values.indexOf(',') > 0) {
-        //     const cookie_percents = percent_values.split(',');
-        //     category_map.map((cat, idx) => {
-        //         if (cookie_percents[idx] && !isNaN(cookie_percents[idx])) {
-        //             to_green_percent[cat] = +cookie_percents[idx].trim();
-        //         }
-        //     });
-        // }
+    const local_storage_server_url = window.localStorage.getItem('config.server_url');
```

Comment:
`window` is a global, is there any difference between using `window.localStorage` and just plain `localStorage`?